Configure

```diff
@@ -135,6 +135,7 @@ my @gcc_devteam_warn = qw(
     -DPEDANTIC -pedantic -Wno-long-long
     -Wall
     -Wextra
+    -Wno-unused-function
```

Comment:
Does this actually make a tangible difference for your purposes?  As far as I can tell, it wouldn't warn us of internal static functions that are unused, and I've found that warning (turned error) useful many times over, so I would rather not have that warning turned off.

Reply:
I've removed it.

Reply:
Somehow the previous discussion on this PR had given me the impression that the idea was going to be to set `-Wunused-function` (not `-Wno-unused-function`).  Maybe I missed some messages...